src/v3/src/components/InputText/InputText.tsx

```diff
@@ -35,23 +38,31 @@ const InputText: UISchemaElementComponent<{
       messages = {},
       name,
     },
+    validate,
   } = uischema.options;
   const messageI18nObj = (messages.value || [])[0]?.i18n;
   const error = t(messageI18nObj?.key, messageI18nObj?.params);
 
   const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
     onChangeHandler(e.target.value);
+    setFieldError(validate?.(e.target.value));
+  };
+
+  const handleConfirmPasswordBlur = () => {
+    setIsTouched(true);
+    setFieldError(t(validate?.(value as string) ?? ''));
```

Comment:
This is handling the onChange/onBlur (a.k.a. real-time feedback) client-side validation. Up to you if you want to move forward with this approach, I thought showing the amount of work required to make this work would be nice to see.